### PR TITLE
[TACHYON-468] Tachyon worker fails to start when under file system is secured HDFS

### PR DESCRIPTION
**Pull request by carsonwang**
https://tachyon.atlassian.net/browse/TACHYON-468

The worker will access the under file system when initializing the WorkerStorage. If the under file system is secured HDFS, it will fail as it has not yet logined.

### CODE REVIEW

File: servers/src/main/java/tachyon/worker/TachyonWorker.java [hunk 1]
```diff
@@ -208,6 +208,15 @@ private TachyonWorker(InetSocketAddress masterAddress, InetSocketAddress workerA
     }
     mWorkerAddress =
         new NetAddress(workerAddress.getAddress().getCanonicalHostName(), mPort, mDataPort);
+
+    // Connect to UFS before initializing the workerStorage.
+    try {
+      connectToUFS();
+    } catch (IOException ioe) {
+      LOG.error(ioe.getMessage(), ioe);
```
Comment (calvinjia) [inline servers/src/main/java/tachyon/worker/TachyonWorker.java]: Maybe change the string portion of this error to `Worker @ hostname failed to connect to the under file system`?

Comment (carsonwang) [inline servers/src/main/java/tachyon/worker/TachyonWorker.java]: Sounds good. I'll update this.

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM. Thanks for the patch

**Pull request merged**